cilium-cli/connectivity/tests/errors.go

```diff
@@ -328,4 +328,6 @@ var (
 	knownIssueWireguardCollision = regexMatcher{regexp.MustCompile("Cannot forward proxied DNS lookup.*:51871.*bind: address already in use")} // from: https://github.com/cilium/cilium/issues/30901
 	// Cf. https://github.com/cilium/cilium/issues/35803
 	endpointMapDeleteFailed = regexMatcher{regexp.MustCompile(`Ignoring error while deleting endpoint.*from map cilium_\w+: delete: key does not exist`)}
+	// envoyTLSWarning is the legitimate warning log for negative TLS SNI test case
+	envoyTLSWarning = regexMatcher{regexp.MustCompile("cilium.tls_wrapper: Could not get server TLS context for pod.*on destination IP.*port 443 sni.*and raw socket is not allowed")}
```

Comment:
Wonder if we could ignore this warning only for the test case? Maybe by including `cilium.io` in the regex?

Reply:
yes, it could be done 👍